### ISSUE TITLE
Customer Creation Supported?

### ISSUE THREAD

**Issue by amenk** [LimeSoda/LimeSoda_SampleDataGenerator#10]
Does it already do this?

**Comment by mzeis** [comment]
No, this isn't supported yet. As mentioned in #11: it may be better to start working on this when v0.2 is released.

**Comment by philwinkle** [comment]
@amenk FYI - customer creation is available via https://github.com/punkstar/magefaker
